deploy/crds/interconnectedcloud_v1alpha1_qdrouterd_crd.yaml

```diff
@@ -136,6 +169,28 @@ spec:
                   - port
                 type: object
               type: array
+            interEdgeConnectors:
```

Comment:
Minor issue, but the name doesn't seem quite right to me. Inter-edge sounds like it is between edges. I think I'd prefer just edgeConnectors (and edgeListeners below) though that is not the nicest name either.